docs/tutorial/part-one/index.md

```diff
@@ -150,7 +150,7 @@ First create the link to the new page.
 To do that, import the `<Link>` component from the `gatsby-link` package that
 was installed along with the starter.
 
-Unlike the normal HTML `<a>` element, our `Link` component uses `to` for
+Unlike the normal HTML `<a>` element, Gatsby's `Link` component uses `to` for
```

Comment:
I don't know if the Link component is specific to Gatsby. It might be specific just to React? @KyleAMathews?

Reply:
Nope, it is Gatsby specific so this is definitely a helpful tweak!